src/cmds/qsub.c

```diff
@@ -551,26 +617,22 @@ expand_varlist(char *varlist)
 			strcat(v_value1, vn);
 			strcat(v_value1, "=");
 			if (copy_env_value(v_value1, vv, 0) == NULL) {
-				fprintf(stderr,
-					"qsub: cannot send environment with the job\n");
+				fprintf(stderr, "qsub: cannot send environment with the job\n");
 				goto expand_varlist_err;
 			}
 		}
 
 		p1 = comma_token(NULL);
 	}
-	if (v_value2 != NULL) {
+	if (v_value2 != NULL)
 		free(v_value2);
-	}
 	return (v_value1);
 
 expand_varlist_err:
-	if (v_value1 != NULL) {
+	if (v_value1 != NULL)
 		free(v_value1);
-	}
-	if (v_value2 != NULL) {
+	if (v_value2 != NULL)
```

Comment:
No need to check whether the value is non-NULL before calling free(). Just call free() without checking.